tools/DocsLinter/Program.cs

```diff
@@ -231,7 +223,20 @@ private static void LogLinkWarning(string markdownFilePath, LinkBase link, strin
         }
 
         /// <summary>
-        /// A helper function the collects all Markdown files from a list of file paths.
+        /// A helper function to log exceptions when they are caught.
+        /// </summary>
+        /// <param name="ex">The exception that was caught</param>
+        private static void LogException(Exception ex)
+        {
+            Console.Error.WriteLine(ex.ToString());
```

Comment:
Print it in red too? Or is that too much?

Reply:
Can definitely do that, I love me some console colours